src/h5web/visualizations/heatmap/ColorMapSelector.tsx

```diff
@@ -27,29 +55,45 @@ function ColorMapSelector(): JSX.Element {
     shallow,
   ]);
 
-  const gradientStyles = {
+  const customStyles = {
     option: (
-      selectStyles: CSSProperties,
-      { data: { label } }: { data: { label: ColorMap } }
+      provided: CSSProperties,
+      { isSelected }: { isSelected: boolean }
+    ) =>
+      isSelected
+        ? {
+            border: 'solid 1px #1b998b',
+          }
+        : {},
```

Comment:
`provided` styles are not used as the styling is given by the custom component `Option`

Reply:
This is no longer the case.